lib/masker/parsers/postgres.rb

```diff
@@ -0,0 +1,50 @@
+module Masker
+  module Parsers
+    class Postgres
```

Comment:
the issue with having this pattern is that only relational databases have `ids`, `tables`, `columns` etc.

so methods like `ids_to_mask`, `missing_tables`, `missing_columns` are only going to work in few cases, the parser public api needs to be much more abstract

Reply:
yeah finding it difficult to make this abstraction work appropriately. i don't know enough about nosql to properly name things, but im pretty sure mongo has a concept of an id for their documents. 

i don't think `parsers` is the correct place to look for the abstractions. the `adapters` are the important abstractions, which right now only need to conform to `mask`. i'm trying to foresee what other methods might make sense on the `adapter` interface other than `mask` but having a difficult time. maybe something like `size` which returns the database size? i'm not sure what else clients would want to send as a message other than `mask`.